techniques/css/C43.html

```diff
@@ -109,19 +109,19 @@ <h2 class="box-h box-h-icon">About this Technique</h2>
 		
 		
 		<section id="description">
-			<h2>Description</h2>
-			<p>The objective of this technique is to ensure that user interface components (for example: links, buttons, and form fields) that are initially completely obscured by a fixed-position component can still be accessed by users. In this example, this is achieved using CSS <code class="language-css">padding</code> and <code class="language-css">scroll-padding</code> properties to create space underneath the site footer and allow the link in the footer to scroll into view when it is focused with a keyboard.</p>
+			<h2>解説</h2>
+			<p>このテクニックの目的は、固定位置コンポーネントによって初めは完全に隠れているユーザインタフェース コンポーネント（例: リンク、ボタン、フォームフィールド）に、利用者が確実にアクセスできるようにすることである。この例では、CSS の <code class="language-css">padding</code> プロパティ及び <code class="language-css">scroll-padding</code> プロパティを使用して、サイトフッターの下にスペースを作成し、キーボードでフォーカスされたときにフッター内のリンクがスクロールして表示されるようにすることで、この目的を実現している。</p>
 		</section>
 		<section id="examples">
-			<h2>Examples</h2>
+			<h2>事例</h2>
 
 			<section class="example" id="example-1-using-css-scroll-padding-to-un-obscure-content">
-				<h3>Example 1: Using CSS <code class="language-css">scroll-padding</code> to un-obscure content</h3>
-				<p>This example shows a situation where there is a fixed-position banner at the bottom of the screen that is covering up the site footer, which contains a link. This type of fixed-position banner is a common design for cookie-consent banners.</p>
+				<h3>事例 1: CSS <code class="language-css">scroll-padding</code> を用いてコンテンツを隠さないようにする</h3>
+				<p>この例では、画面下部に固定位置バナーが存在し、リンクを含むサイトフッターが隠れている状況を示している。このタイプの固定位置バナーは、Cookie 同意バナーでよく使用されるデザインである。</p>
         <aside class="note">
-          <p>This example demonstrates a technique to un-obscure content underneath a fixed-position banner. To prevent the page becoming unusable at smaller screen sizes, the banner becomes un-fixed. To observe the <code class="language-css">scroll-padding</code> effect you will need to use a viewport over 800px wide.</p>
+          <p>この例では、固定位置バナーの下にあるコンテンツを隠さないようにするテクニックを示している。小さい画面サイズでページが使いにくくなるのを防ぐため、バナーは固定されていない。<code class="language-css">scroll-padding</code> の効果を確認するには、幅 800px 以上のビューポートを使用する必要がある。</p>
```

Comment:
becomes un-fixed は「固定されていない」というより「固定されていない状態になる」、「固定されなくなる」のではないでしょうか。

```suggestion
          <p>この例では、固定位置バナーの下にあるコンテンツを隠さないようにするテクニックを示している。小さい画面サイズではページが使いにくくなるのを防ぐため、バナーは固定されなくなる。<code class="language-css">scroll-padding</code> の効果を確認するには、幅 800px 以上のビューポートを使用する必要がある。</p>
```